mlflow/projects/__init__.py

```diff
@@ -204,13 +184,13 @@ def _run(uri, experiment_id, entry_point="main", version=None, parameters=None,
 
     supported_backends = ["local", "databricks", "kubernetes"]
     raise ExecutionException("Got unsupported execution mode %s. Supported "
-                             "values: %s" % (backend, supported_backends))
+                             "values: %s" % (backend_name, supported_backends))
 
 
 def run(uri, entry_point="main", version=None, parameters=None,
         experiment_name=None, experiment_id=None,
         backend=None, backend_config=None, use_conda=True,
-        storage_dir=None, synchronous=True, run_id=None):
+        storage_dir=None, synchronous=True, run_id=None, tracking_store_uri=None):
```

Comment:
```suggestion
        storage_dir=None, synchronous=True, run_id=None):
```

Sorry for the confusion on this one, I don't think we should necessarily expose a `tracking_store_uri` argument to the top-level `mlflow.projects.run` API, or at least not in this PR :). Seems reasonable to have it in the backend though.

Reply:
Done